src/variables.tf

```diff
@@ -225,12 +225,25 @@ variable "ip_nodo" {
   description = "Nodo pagamenti ip"
 }
 
-variable "nodo_pagamenti_oncloud_url" {
+variable "lb_aks" {
   type        = string
-  description = "Nodo pagamenti url oncloud"
+  description = "Load balancer AKS Nexi/SIA"
   default     = "http://"
```

Comment:
mmmm this default it's a bit confusing don't you?

Reply:
ok,  I fix it